azure-pipelines.yml

```diff
@@ -10,6 +10,11 @@ jobs:
   steps:
   - script: python3 -m pip install --upgrade pip setuptools --user
     displayName: 'Install python tools'
+  - script: |
```

Comment:
why we need to merge the master to dev-mc?

Reply:
This is from dev to master